tests/e2e/specs/edit-story/example.js

```diff
@@ -17,12 +17,57 @@
 /**
  * Internal dependencies
  */
-import { createNewStory } from '../../utils';
+import { createNewStory, exportData, popStats } from '../../utils';
 
-describe('Example Spec', () => {
-  it('should be able to create a blank story', async () => {
+// To debug:
+// 1. Set `PUPPETEER_DEVTOOLS=true`
+// 2. Insert `await jestPuppeteer.debug()` in the test being debugged.
+
+describe('Selection with new element', () => {
+  let safezoneBox;
+
+  beforeEach(async () => {
     await createNewStory();
 
-    await expect(page).toMatchElement('input[placeholder="Add title"]');
+    // Click on a first media image.
+    await expect(page).toClick('[data-testid="mediaElement"] img');
```

Comment:
Dumb question: where is `page` defined?

Reply:
It's a global. There are few others here: https://github.com/smooth-code/jest-puppeteer